lib/home_page.dart

```diff
@@ -1,4 +1,20 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:intl/intl.dart';
+import 'package:supabase_flutter/supabase_flutter.dart';
+
+import '../../util/buttons/button.dart';
+import 'account/widgets/avatar.dart';
+import 'drives/pages/create_drive_page.dart';
+import 'drives/pages/drive_detail_page.dart';
+import 'rides/pages/ride_detail_page.dart';
+import 'rides/pages/search_ride_page.dart';
+import 'util/chat/models/message.dart';
+import 'util/chat/pages/chat_page.dart';
+import 'util/model.dart';
+import 'util/parse_helper.dart';
+import 'util/ride_event.dart';
+import 'util/supabase.dart';
 
 class HomePage extends StatefulWidget {
```

Comment:
I had a null reference exception earlier today. I couldn't even open the app. Unfortunately, I can't reproduce it right now, and the stack trace was more than 1000 lines long, making it impossible for me to see the actual exception because the shell cuts off after 1000 lines (bad shell). But I think it was visible to everyone at the time, including AG 😬 Could you please check that every ! operator is used with merit? Otherwise, if this doesn't happen again, maybe the supabaseClient wasn't set and we'll rule it as a freak occurence